submissions/BogdanGapon/HTML & CSS popup/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="header-nav">
+          <ul class="header-nav__list">
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link">Mail</a>
+            </li>
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link" target="_blank">Images</a>
+            </li>
+            <li class="header-nav__item popup-item">
+              <label class="custom-checkbox" for="check"></label>
+              <input
+                class="checkbox visually-hidden"
+                type="checkbox"
+                id="check"
+              />
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <div class="popup-flex-wrapper">
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/apps.png"
+                      alt="Colorful circle"
+                      width="45"
+                      height="45"
+                    />Colorful circle</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/blogs.png"
+                      alt="Blog"
+                      width="45"
+                      height="45"
+                    />Blog</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/calendar.png"
+                      alt="Calendar"
+                      width="45"
+                      height="45"
+                    />Calendar</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/chat.png"
+                      alt="Chat"
```

Comment:
Please, delete all alt values in img tag in nav or popUP. If image is used for decoration and don't share any information to user, its alt attribute should be ampty without any symbols or spacing.